src/log.h

```diff
@@ -48,15 +65,21 @@ namespace librealsense
             defaultConf.setGlobally(el::ConfigurationType::LogFlushThreshold, "10");
             defaultConf.setGlobally(el::ConfigurationType::Format, " %datetime{%d/%M %H:%m:%s,%g} %level [%thread] (%fbase:%line) %msg");
 
+            // Have the logger immediately write to a file, so we catch any message right away
+            // Note this may have performance implications -- better to use callbacks!
+            el::Loggers::addFlag( el::LoggingFlag::ImmediateFlush );
```

Comment:
Need to check / discuss

Reply:
As discussed, will try to move to the individual tests